poi.config.js

```diff
@@ -101,6 +101,13 @@ module.exports = {
       .plugin('dotenv')
       .use(require.resolve('dotenv-webpack'));
 
+    config
+      .module
+      .rule('eslint')
+      .use('eslint-loader')
+        .loader('eslint-loader')
+        .tap(options => Object.assign(options, { ignore: true }));
```

Comment:
If I understand code of `eslint-loader` correctly this gets passed down to `eslint`'s `CLIEngine`: https://github.com/webpack-contrib/eslint-loader/blob/2a72daa5b78e3ab0a1d9861210f21e7bb4440948/index.js#L22

That being said, docs on following page: https://eslint.org/docs/developer-guide/nodejs-api#cliengine clearly say that:
>ignore - False disables use of .eslintignore, ignorePath and ignorePattern (default: true). 

So what does this do really? 🤔

Reply:
In build process for production, for some reason, `ignore` would be resolved to `false`, thus _ignoring_ `ignorePath` and/or `ignorePattern`.

Reply:
Hmm, this thing http://npmfs.com/package/@poi/plugin-eslint/12.0.0/index.js does not toggle it so what does?

Reply:
I know, that's what was baffling me as well.
After more than an hour of debugging, we couldn't figure out what was happening, we only came to the conclusion that setting `ignore: true` explicitly resolves the issue...

Reply:
I assume you did run `webpack:inspect-config` in prod mode? No luck there?

Reply:
Yes, but the config seemed fine.